internal/api/assets.go

```diff
@@ -211,3 +213,67 @@ func (h handler) GetAsset(w http.ResponseWriter, r *http.Request) {
 
 	respondwith.JSON(w, http.StatusOK, asset)
 }
+
+func (h handler) PostAssetErrorResolved(w http.ResponseWriter, r *http.Request) {
+	httpapi.IdentifyEndpoint(r, "/v1/projects/:id/assets/:type/:uuid/error-resolved")
+
+	projectUUID, token := h.CheckToken(w, r)
+	if token == nil {
+		return
+	}
+	if !token.Require(w, "cluster:access") {
+		return
+	}
+	dbResource := h.LoadResource(w, r, projectUUID, token, false)
+	if dbResource == nil {
+		return
+	}
+	if !token.Require(w, dbResource.AssetType.PolicyRuleForWrite()) {
+		return
+	}
+
+	var queryResult struct {
+		AssetID          int64                      `db:"id"`
+		OperationReason  castellum.OperationReason  `db:"reason"`
+		OperationOutcome castellum.OperationOutcome `db:"outcome"`
+	}
+
+	err := h.DB.SelectOne(&queryResult,
+		`SELECT a.id, fo.reason, fo.outcome
+		 FROM assets a
+		 LEFT JOIN finished_operations fo ON a.id = fo.asset_id
+		 WHERE a.resource_id = $1 AND a.uuid = $2
+		 ORDER BY fo.finished_at DESC LIMIT 1`,
+		dbResource.ID, mux.Vars(r)["asset_uuid"])
+
+	if errors.Is(err, sql.ErrNoRows) {
+		respondWithNotFound(w)
+		return
+	}
+	if respondwith.ErrorText(w, err) {
+		return
+	}
+	if queryResult.OperationOutcome != castellum.OperationOutcomeErrored {
+		respondWithConflict(w)
```

Comment:
There is already a library function for printing a text message:
```suggestion
		http.Error(w, "409 Conflict", http.StatusConflict)
```
Also, I would like to have a more specific error message here.

Regarding the existing functions, no idea why I built it this way originally...